source/gui/settingsDialogs.py

```diff
@@ -1829,6 +1861,10 @@ def makeSettings(self, settingsSizer):
 		)
 		self._appendSpeechModesList(settingsSizerHelper)
 
+	def onAutoLanguageSwitchingChange(self, evt: wx.CommandEvent):
+		self.reportLanguageCheckbox.Enabled = self.autoLanguageSwitchingCheckbox.IsChecked()
+		self.reportNotSupportedLanguageCombo.Enabled = self.autoLanguageSwitchingCheckbox.IsChecked()
```

Comment:
please don't set enabled like this, instead
```suggestion
		self.reportNotSupportedLanguageCombo.Enable(self.autoLanguageSwitchingCheckbox.IsChecked())
```